lib/diff/file_source.rb

```diff
@@ -0,0 +1,22 @@
+module Diff
+  class FileSource
+    def self.read(path)
+      new(File.read(path))
```

Comment:
There is also `File.read_lines` but it was just as easy to use `File.read` and I didn't have to strip the newlines that would have been on each line it read.